util/PostgresMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -32,13 +32,13 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("text");
 
                     b.Property<DateTime>("CreationDate")
-                        .HasColumnType("timestamp with time zone");
+                        .HasColumnType("timestamp without time zone");
```

Comment:
@Hinton Looks like this was a swapped to `with time zone` with a change pulled from master and originally was `without time zone`.  So now it's getting set back to `without time zone`

This all has to do with 
https://github.com/bitwarden/server/blob/f3085f531ddb61f0bdd5193df5b13c2a28a16bb7/util/PostgresMigrations/Factories.cs#L27-L29

Which hasn't been merged into master for migrations, but is set for runtime EF configuration. 

I'm good with how it is in this PR, just want to call attention to these values being swapped and some potential future confusion.

Reply:
Sounds like we should fix this for migrations in a separate PR and merge that before to avoid switching timestamp formats.

Reply:
We already did in sm/secrets-feature. 

A change that was merged in from master made a postgres migration without the applied fix.

Reply:
Ah, we should cherry pick those changes to master.

Reply:
https://github.com/bitwarden/server/pull/2238